schemas/cloudflare/account-1.yml

```diff
@@ -0,0 +1,44 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /cloudflare/account-1.yml
+  name:
+    type: string
+  description:
+    type: string
+  providerVersion:
+    type: string
+  accountOwners:
+    type: array
+    items:
+      type: object
+      properties:
+        name:
+          type: string
+        email:
+          type: string
+          format: email
+      required:
+      - name
+      - email
+  apiCredentials:
+    "$ref": "/common-1.json#/definitions/vaultSecret"
+  terraformStateAccount:
```

Comment:
is there a reason to use the account and not the state, like is done in accounts?

Reply:
The state files don't link back to an account, but accounts do link to a state file.

All other terraform integrations that we have implicitly use the same AWS account which is set as the provisioner. In the case of non-AWS resources (Cloudflare) we can't do that and we must somehow specify what AWS account we store the state in

Did I miss something that could make this better?

Reply:
i think you can crossref `/dependencies/terraform-state-1.yml` instead of `/aws/account-1.yml`?

Reply:
but `/dependencies/terraform-state-1.yml` does not reference an account, that is my point. so how do I know what account to use?

referencing `/aws/account-1.yml` I get both an account and its terraform state settings

Reply:
ok, understood.

if the automationToken and some of the terraform configuration was in the terraformState it would make sense.

sorry for the noise!